### DISCUSSION TITLE
Are there any open-source projects that utilizes this library ?

### DISCUSSION THREAD

**Discussion by mziadd** [ngageoint/geopackage-ios#50] [Q&A] [answered]
I have a novice experience in iOS development using swift, my idea is strengthening my skills by learning and customization existing apps to my needs.

**Comment by bosborn** [comment, marked as answer]
These are our open source libraries that utilize this library.

From the README:

#### Example Apps ####

The following projects demonstrate usage of geopackage-ios in a host app.

##### GeoPackage MapCache #####

The [GeoPackage MapCache](https://github.com/ngageoint/geopackage-mapcache-ios) app provides an extensive standalone example on how to use the SDK.

##### MAGE #####

The [Mobile Awareness GEOINT Environment (MAGE)](https://github.com/ngageoint/mage-ios) app provides mobile situational awareness capabilities. It [uses the SDK](https://github.com/ngageoint/mage-ios/search?q=GeoPackage&type=Code) to provide GeoPackage functionality.
